src/main/java/org/fundacionjala/coding/maria/Kata10.java

```diff
@@ -0,0 +1,22 @@
+package org.fundacionjala.coding.maria;
+
+import java.util.Arrays;
+import java.util.StringJoiner;
+
+/**
+ * kata 10.
+ */
+public class Kata10 {
+    private static final String BLANK_SPACE = " ";
+
+    /**
+     * @param numbers numbers
+     * @return hig and low number
+     */
+    public String highAndLow(String numbers) {
+        StringJoiner test = new StringJoiner(BLANK_SPACE);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "test". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "test" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)